datanode/gateway/graphql/schema.graphql

```diff
@@ -2310,6 +2310,54 @@ type Trade {
   sellerAuctionBatch: Int
 }
 
+"A trade on Vega, the result of two orders being 'matched' in the market"
+type TradeUpdate {
+  "The hash of the trade data"
+  id: ID!
+
+  "The market the trade occurred on"
+  marketId: ID!
+
+  "The order that bought"
+  buyOrder: String!
+
+  "The order that sold"
+  sellOrder: String!
+
+  "The party that bought"
+  buyerId: ID!
+
+  "The party that sold"
+  sellerId: ID!
+
+  "The aggressor indicates whether this trade was related to a BUY or SELL"
+  aggressor: Side!
+
+  "The price of the trade (probably initially the passive order price, other determination algorithms are possible though) (uint64)"
+  price: String!
+
+  "The number of contracts trades, will always be <= the remaining size of both orders immediately before the trade (uint64)"
```

Comment:
```suggestion
  "The number of units traded, will always be <= the remaining size of both orders immediately before the trade (uint64)"
```
I want to get confirmation on the use of "units" here but I'm sure that we shouldn't be using "contracts"

Reply:
This needs to be confirmed with @barnabee .

Reply:
Confirmed -  please go ahead with units @ValentinTrinque @jeremyletang